cryptographic_estimators/UOVEstimator/UOVAlgorithms/kipnis_shamir.py

```diff
@@ -20,6 +20,7 @@
 from ..uov_problem import UOVProblem
 from math import log2
 from sage.arith.misc import is_power_of_two
```

Comment:
Please use the python function `is_power_of_two` which is in `helper.py`

Reply:
I deleted that import because it wasn't being used